### PR TITLE
ImageJ Strokewidth update

### PR DESCRIPTION
**Pull request by bramalingam**
@pwalczysko : This PR should fix the Stroke-width bug suggested here : https://github.com/openmicroscopy/bioformats/pull/1671#issuecomment-82458999

### CODE REVIEW

File: components/bio-formats-plugins/src/loci/plugins/util/ROIHandler.java [hunk 1]
```diff
@@ -36,6 +36,7 @@
 import ij.gui.PointRoi;
 import ij.gui.PolygonRoi;
 import ij.gui.Roi;
+import ij.gui.RoiProperties;
```
Comment (melissalinkert) [inline components/bio-formats-plugins/src/loci/plugins/util/ROIHandler.java]: This import could likely be removed.

**Comment by pwalczysko** [comment]
Works as expected. Fiji as well as ImageJ.
Ready to merge.

**Comment by pwalczysko** [comment]
(note that this is a bugfix which should make it to 5.1.0)

**Comment by bramalingam** [comment]
--no-rebase

**Pull request merged**